sciplot/specs/HistogramStyleSpecs.hpp

```diff
@@ -127,8 +127,8 @@ inline auto HistogramStyleSpecs::errorBarsWithLineWidth(double value) -> Histogr
 
 inline auto HistogramStyleSpecs::repr() const -> std::string
 {
-    const auto supports_gap = (m_type == "clustered" || m_type == "errorbars");
-    const auto supports_linewidth = (m_type == "errorbars");
+    [[maybe_unused]] const auto supports_gap = (m_type == "clustered" || m_type == "errorbars");
+    [[maybe_unused]] const auto supports_linewidth = (m_type == "errorbars");
```

Comment:
I am not sure why these indeed unused variables are there – maybe for debugging reasons? Since I don't know the rationale I decided to add a `[[maybe_unused]]` attribute instead of deleting it to resolve the warning.